mimic-navigation/navigation.md

```diff
@@ -0,0 +1,97 @@
+# Navigating Mimic
+
+Mimic is a single-window application. The interface is divided into four main parts:
+
+- **Action Bar**
+- **Routes**
+- **Logs**
+- **Status bar**
+
+Every part of the application was designed to give you enough space for showing information.
+
+## Action Bar
+
+![alt text](../Images/actionbar.png 'Action bar')
+
+On the Action Bar, the following options are available:
+
+- **Projects** Button - Create a new project, switch between projects.
+
+- **Edit** Icon - Edit the project name.
+
+- **Start Server** Button - Start the server, stop the server.
+
+- **Plus(+)** Button - Add a route to the project.
```

Comment:
[style] We need to decide whether we talk about _routes_ or _endpoints_, because currently we mix those terms here. I think it'd be better to settle down on the _endpoint_, but I'm not sure…

@marras What do you think?

Reply:
@marras @sharnik but in app we stays with Routes as a section title?

Reply:
In my opinion it would be best if we just changed it to "Endpoints" everywhere. @sharnik is that ok?

Reply:
@marras you mean in app code too?

Reply:
Yes, that's exactly what I meant.

Reply:
I changed routes for endpoints but I leave section name like in the app at the moment (Routes) :)

Reply:
I'll create a ticket in the app to make sure we use _Endpoints_ everywhere.